config/initializers/devise.rb

```diff
@@ -287,6 +287,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :linkedin, ENV['LINKEDIN_KEY'], ENV['LINKEDIN_SECRET'],
```

Comment:
Do you think that it may cause some problems if this code is executed then `LINKEDIN_SIGNIN_ENABLED` is not enabled?